services/api/service.go

```diff
@@ -1315,6 +1317,13 @@ func (api *RelayAPI) handleSubmitNewBlock(w http.ResponseWriter, req *http.Reque
 		return
 	}
 
+	// save execution payload to memcached as secondary backup to Redis
+	err = api.memcached.SaveExecutionPayload(payload.Slot(), payload.ProposerPubkey(), payload.BlockHash(), getPayloadResponse)
```

Comment:
can api.memcached not be nil?

Reply:
It can - but since the method receiver returns early when caller is nil we can invoke this method without panicking 

Open to other approaches, more discussion found [here](https://github.com/flashbots/mev-boost-relay/pull/303#discussion_r1130579958)

Reply:
Updated, see: [link](https://github.com/flashbots/mev-boost-relay/pull/303#discussion_r1130896983)